src/openrct2/actions/StaffHireNewAction.hpp

```diff
@@ -262,7 +262,7 @@ DEFINE_GAME_ACTION(StaffHireNewAction, GAME_COMMAND_HIRE_NEW_STAFF_MEMBER, Staff
 
             for (int32_t newStaffIndex = 0; newStaffIndex < STAFF_PATROL_AREA_SIZE; newStaffIndex++)
```

Comment:
Change this variable to `i` to remove future confusion.

Reply:
OK those are both done, cheers :)